server/views/serviceProviderReferrals/reviewActionPlan.njk

```diff
@@ -0,0 +1,43 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+
+{% extends "./actionPlan/actionPlanFormTemplate.njk" %}
+
+{% block formSection %}
+  <p class="govuk-body-m">Review the action plan before submitting it to the service user’s probation practitioner for approval. The first version of the action plan must be submitted within 5 working days from the initial assessment.</p>
+
+  <h3 class="govuk-heading-m">
+    Outcomes and activities
+  </h3>
+
+  {% for desiredOutcome in presenter.desiredOutcomes %}
+    {# This could do with some design eyes on the heading sizes #}
+      <h4 class="govuk-heading-s">
+        Desired outcome {{ loop.index }}
+      </h4>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>{{ desiredOutcome.description }}</li>
+      </ul>
+
+      {% for activity in desiredOutcome.activities %}
+        <h5 class="govuk-body govuk-!-font-weight-bold">
+          Activity {{ loop.index }}
+        </h5>
+        <p class="govuk-body">{{ activity.text }}</p>
+      {% endfor %}
+  {% endfor %}
+
+  <h3 class="govuk-heading-m">
+    Suggested number of sessions for the action plan
+  </h3>
+
+  <p class="govuk-body">
+  Suggested number of sessions: {{ presenter.text.numberOfSessions }}
```

Comment:
This line should be indented [sorry]

Reply:
This seems to be intentional on the part of my editor - it seems to not indent the contents of a `<p>` tag, for… reasons…

Could we leave it as is? It would be lovely to find some way of formatting these files consistently across the team though.

Reply:
Yeah, HTML and nunjucks are seemingly impossible to format...